docs/usage/quickstart.rst

```diff
@@ -152,9 +152,10 @@ Access to dashboard, Grafana and Kibana
 Once the cluster is running, you can access the `Kubernetes dashboard`_,
 Grafana_ metrics and Kibana_ logs from your browser.
 
-The Kubernetes dashboard is available at :samp:`https://{master-ip}:6443/ui`,
-where *master-ip* should be replaced by the IP-address of one of the hosts in
-the *kube-master* group.
+To access the Kubernetes dashboard, first create a secure tunnel into your
+cluster by running ``kubectl proxy``. Then, while the tunnel is up and running,
+access the dashboard at
+http://localhost:8001/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:/proxy/.
```

Comment:
I don't know if we want to add this as well but the dashboard was also accessible via network IP on my deployment.
```https://<node-ip-address>:6443/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:/proxy/#!/overview?namespace=default```

Otherwise LGTM

Reply:
Indeed, but then you have to put your `Basic` auth login/password in the URL, otherwise you get an `Access denied`, which kinda sucks.

@Zempashi and I already discussed to remove the `Ingress`es we set up for Kibana, Grafana,.... and enforce access through `kubectl proxy` to them as well, for security reasons (currently they're wide open!). Also accessing the dashboard through said tunnel then makes sense.

Reply:
Yeah that sounds like the best(safest) option overall for this project.